pkg/controllers/provisioning/provisioner.go

```diff
@@ -53,6 +54,8 @@ import (
 	"sigs.k8s.io/karpenter/pkg/utils/pretty"
 )
 
+var GlobalDefaultTerminationGracePeriod *metav1.Duration = nil
```

Comment:
nit: want to leave a small comment here on the use case?

Reply:
Sure, I can add that